content/en/agent/logs/advanced_log_collection.md

```diff
@@ -443,6 +443,92 @@ More examples:
 | 2020-10-27 05:10:49.657  | `\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d{3}` |
 | {"date": "2018-01-02"    | `\{"date": "\d{4}-\d{2}-\d{2}`                |
 
+### Automatic Multi-line aggregation
+With Agent 7.32+ `auto_multi_line_detection` can be enabled which allows the Agent to detect [common multi-line patterns](https://github.com/DataDog/datadog-agent/blob/main/pkg/logs/decoder/auto_multiline_handler.go#L195) automatically. This feature is currently in beta. 
```

Comment:
We use reference link syntax for our translation pipeline, please add this as a new number at the bottom. (It's okay to have numbers out of order, our link checker will fix it the next time any commit is made.)

Also, we usually have a specific colored/styled banner for beta notices, did you want to use one of those here?

Reply:
Thanks for the tip!
Yeah we can use the banner, do you have an example I can look at? I'm not really sure what I'm looking for.

Reply:
Yeah, it's just a small bit of html, so you'd wrap it in a div with these classes:

```html
<div class="alert alert-warning">Text</div>
```
and it'll put whatever sentence you have in there in a yellow banner/div box. If they need to request access, this is also where you'd add the link to the form (using html instead of markdown inside the div tag) or instructions on how to do so (sometimes all you have to do is talk to your customer success manager). One example is on the watchdog page, we have this beta warning:

```html
## Root Cause Analysis for APM (beta)

<div class="alert alert-warning">
Watchdog Root Cause Analysis (RCA) is currently in beta. <a href="https://docs.google.com/forms/d/1gKdjVslrxMv_St7sIH7c47FJuf_hKvKAt_G9SXnE4Nw/edit?ts=5fbd5390&gxids=7628">Use this form</a> to request access.
</div>
```

Reply:
Thanks! I added the beta banner! Let me know if I should change it in any way :)